orthw

```diff
@@ -207,6 +207,7 @@ analyze_in_docker() {
     -it \
     $mount_netrc_option \
     -v $ort_home:/ort \
+    -v $HOME/.ssh:/home/ort/.ssh \
```

Comment:
Commit message:
- The wording sounds as if `git` in general could only be used with `ssh` as a transport, which is not correct. I'd simply say "In case Git repositories need to be cloned via SSH, provide access to the host's SSH config" or similar.
- Typo in "hosts" -> "host's" (already corrected in my example sentence above.

Reply:
The new commit message lacks a trailing dot (you copied & pasted a bit too literally 😉).